contribute/contribute_rockons.rst

```diff
@@ -64,6 +103,98 @@ guidelines can help:
 * Finally, the image size is another helpful factor, with smaller images being
   usually favored.
 
+Environment setup
+-----------------
+
+Go to the `rockon-registry repo <https://github.com/rockstor/rockon-registry>`_
+and click on the <b>Fork</b> button. This will fork the repository into your
+profile which serves as your private git remote called origin. The next few
```

Comment:
```suggestion
profile which serves as your personal git remote called origin. The next few
```
I had thought a default fork in this context was non private, ergo 'personal' suggested word change.